src/Api/Models/Request/Organizations/OrganizationSmSubscriptionUpdateRequestModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Bit.Core.Models.Business;
+
+namespace Bit.Api.Models.Request.Organizations;
+
+public class OrganizationSmSubscriptionUpdateRequestModel
+{
+    [Required]
+    public int SeatAdjustment { get; set; }
+    public int? MaxAutoscaleSeats { get; set; }
+
+    public int? ServiceAccountAdjustment { get; set; }
```

Comment:
I think this will always get a value, even if 0. (Please double check this though)

Reply:
resolved